Makefile

```diff
@@ -24,6 +24,8 @@ backend-typing:
 install-python-dependencies:
 	pip install -e .
 	pip install -r requirements-test.txt
+	pip uninstall -y sentry-arroyo
+	pip install git+https://github.com/getsentry/arroyo@9bbb9195698c25a0ce687cd8a00e433bc8da5d3d#egg=sentry-arroyo
```

Comment:
Should be removed before merging. (New Arroyo version would be added to requirements.py instead)